.xmos_ignore_source_check

```diff
@@ -0,0 +1,4 @@
+/Cyberon/lib
```

Comment:
guess we need license sign off before this line

Reply:
Line removed!